screens/setting/SettingsScreen.js

```diff
@@ -171,8 +174,57 @@ class SettingsScreen extends Component {
                         />
                     </View>
                 </View>
-                <SafeAreaView style={{ flex: 0, backgroundColor: '#f7f7f7' }} />
-            </>
+                <ActionSheet
+                    closeOnTouchBackdrop={false}
+                    ref={this.actionSheetRef}>
+                    <View
+                        style={{
+                            height: 300,
+                            padding: 40,
+                            borderTopLeftRadius: 8,
+                            borderTopRightRadius: 8,
+                            alignItems: 'center',
+                            backgroundColor: 'white',
+                            justifyContent: 'center'
+                        }}>
+                        <Body style={{ textAlign: 'center' }}>
+                            Do you want to change picked up status of all the
```

Comment:
Need to add this translation to trello

https://trello.com/invite/b/E0JDKi89/2be6b5f4177568b9128e476606f40fd4/olm-translations